scripts/pull_github_pr.sh

```diff
@@ -36,6 +48,52 @@ curl() {
     command curl "${opts[@]}" "$@"
 }
 
+set_jenkins_job() {
+  branch=$(git rev-parse --abbrev-ref HEAD)
+  product=$(cat SCYLLA-VERSION-GEN | grep ^PRODUCT | awk -F'=' '{print $2}')
+
+
+  if [[ "$branch" == "next" ]]; then
+    jenkins_job="scylla-master/job/next"
+  fi
+  if [[ "$branch" == "next-enterprise" ]]; then
+    jenkins_job="scylla-enterprise/job/next"
+  fi
+  if [[ "$branch" == next-* ]]; then
```

Comment:
Please consider using "case" instead of if - it's shorter, nicer, and the globbing is less suspicious:
```
case "$branch" in
next) jenkins_job=scylla-master/job/next;;
next-enterprise) jenkins_job=scylla-enterprise/job/next;;
next*) ...
   ;;;
esac
```

Reply:
fixed